[2025-03-13T12:25:54.417+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_api_to_sqlite.load_api_task scheduled__2025-03-12T00:00:00+00:00 [queued]>
[2025-03-13T12:25:54.426+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_api_to_sqlite.load_api_task scheduled__2025-03-12T00:00:00+00:00 [queued]>
[2025-03-13T12:25:54.427+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-03-13T12:25:54.439+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_api_task> on 2025-03-12 00:00:00+00:00
[2025-03-13T12:25:54.443+0000] {standard_task_runner.py:60} INFO - Started process 294 to run task
[2025-03-13T12:25:54.446+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_api_to_sqlite', 'load_api_task', 'scheduled__2025-03-12T00:00:00+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/etl_api_to_sqlite.py', '--cfg-path', '/tmp/tmpu94l22h8']
[2025-03-13T12:25:54.447+0000] {standard_task_runner.py:88} INFO - Job 22: Subtask load_api_task
[2025-03-13T12:25:54.482+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_api_to_sqlite.load_api_task scheduled__2025-03-12T00:00:00+00:00 [running]> on host 029b78518459
[2025-03-13T12:25:54.541+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_api_to_sqlite' AIRFLOW_CTX_TASK_ID='load_api_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-12T00:00:00+00:00'
[2025-03-13T12:25:54.723+0000] {python.py:201} INFO - Done. Returned value was: API data loaded successfully to SQLite. 70 records inserted in /opt/***/data/etl_database.db
[2025-03-13T12:25:54.737+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_api_to_sqlite, task_id=load_api_task, execution_date=20250312T000000, start_date=20250313T122554, end_date=20250313T122554
[2025-03-13T12:25:54.779+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-13T12:25:54.790+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
